### ISSUE TITLE
Sliders do not need to be activated to be adjusted

### ISSUE THREAD

**Issue by haykam821** [jellysquid3/Sodium#2310]
When a slider is selected using keyboard navigation, using the left and right arrow keys will adjust the slider accordingly despite the slider not being activated. The slider does not have differentiated focus states:

<img width="714" alt="Focused slider in the Sodium video options screen" src="https://github.com/CaffeineMC/sodium-fabric/assets/24855774/9e16c149-e1d6-460c-9d3c-9de700150b5b">

This behavior differs from sliders outside of the Sodium video options screen, which both require focus then activation for the value to be controlled:

<img width="547" alt="Focused slider outside of the Sodium video options screen before activation" src="https://github.com/CaffeineMC/sodium-fabric/assets/24855774/52ebf1d5-badc-46d7-8224-96db6aa6227c"> 
<img width="539" alt="Activated slider outside of the Sodium video options screen" src="https://github.com/CaffeineMC/sodium-fabric/assets/24855774/fe0c4eef-f46e-4ec4-a3c4-e1d3b30ef120">

This issue can be reproduced with Minecraft 1.20.4 and the build of the latest commit as of this issue: 82f4af61fd4ea4c4c5c8e02897202f627d4e68cb.
